Eraware_Dnn_Spa_Ef_Di_Stencil/module.web/src/components/my-item-details/my-item-details.tsx

```
@@ -39,25 +39,24 @@ export class MyItemDetails {
-              confirmMessage={this.resx.deleteItemConfirm || "Are you sure you want to delete this item?"}
-              confirmNoText={this.resx.no || "No"}
-              confirmYesText={this.resx.yes || "Yes"}
+              confirmMessage={this.resx?.deleteItemConfirm || "Are you sure you want to delete this item?"}
+              confirmNoText={this.resx?.no || "No"}
+              confirmYesText={this.resx?.yes || "Yes"}
               onConfirmed={() => this.deleteItem()}
-            >{this.resx.delete || "Delete"}</dnn-button>
+            >{this.resx?.delete || "Delete"}</dnn-button>
             <dnn-modal
-              ref={e => this.modal = e}
+              ref={e => this.modal = e!}
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)